packages/Select/index.tsx

```diff
@@ -329,6 +334,7 @@ export const Select = forwardRef<'input', SelectProps>(
                                   const isItemSelected = isValueSelected(option.value, selected)
                                   return (
                                     <S.Item
+                                      $disabled={option.disabled}
```

Comment:
Could you please tell me why there is a $ in front of disabled?

Reply:
normally is to not be on the dom, only for the styled component

Reply:
it's to prevent `disabled` to be passed to the <li> in the DOM. Otherwise it will have a different behavior.